consensus/src/alerts/mod.rs

```diff
@@ -490,3 +490,367 @@ pub(crate) async fn run<H: Hasher, D: Data, MK: MultiKeychain>(
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
You don't test `alert_confirmed` at all, even though `run` uses it directly.

Reply:
Added in 5d4be267 (thanks to @DamianStraszak for the help)!